modules/comparing-runs.adoc

```diff
@@ -0,0 +1,51 @@
+:_module-type: PROCEDURE
+
+[id='comparing-runs_{context}']
+= Comparing runs
+
+[role='_abstract']
+You can compare up to 10 pipeline runs at one time, and view available parameter, scalar metric, confusion matrix, and receiver operating characteristic (ROC) curve data for all selected runs.
+
+.Prerequisites
+* You have logged in to {productname-long}.
+ifdef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group} ) in OpenShift.
+endif::[]
+* You have previously created a data science project that is available and has a pipeline server.
+* You have imported a pipeline to an active pipeline server.
+* You have created at least 2 pipeline runs.
+
+.Procedure
+. In the {productname-short} dashboard, select *Experiments > Experiments and runs*.
++ 
+The *Experiments* page opens.
```

Comment:
Regarding these step results, such as: 
+
The *Experiments* page opens.

I spoke to Melissa about these, it was suggested to me by Melissa that these step results contravene minimalism standards. I then removed them wherever I could from the DSP modules.

Reply:
Fair, I'll update to remove these result steps.

Reply:
Per your overall review comment, I'll leave these til we have further team discussion.